### ISSUE TITLE
Tutorial screen for minesweeper game does not match the game screen

### ISSUE THREAD

**Issue by sunjunkie** [systers/powerup-iOS#127]
The minesweeper tutorial screen describes 5 vertical stars as the available number of tries left. But the game itself does not show these 5 stars. See attached screenshots
![simulator screen shot aug 14 2017 1 16 59 pm](https://user-images.githubusercontent.com/399376/29291297-fb38aa06-80f7-11e7-9100-08f2e706ec43.png)
![simulator screen shot aug 14 2017 1 18 17 pm](https://user-images.githubusercontent.com/399376/29291302-ff18ef78-80f7-11e7-82f7-cb3092383598.png)

**Comment by justKD** [comment]
It looks like the tutorial screens just need to be updated. I think the "Score" was just moved to the top right in a different format.

I'm looking through the code, and I don't see any reference to the stars mentioned in the current tutorial screens.

**Comment by sunjunkie** [comment]
You're right. We could update the tutorial screen to indicate the number of tries the user can make.

**Comment by SIMRAN1** [comment]
@sunjunkie May I work on this??

**Comment by vatsalkul** [comment]
Hey @sunjunkie I have updated the tutorial screen according to the above discussion. Please have a look and approve me to submit a PR. [Drive Link](https://drive.google.com/file/d/1c8FmbSC1sqrAsTI9ziwdzpvfdDw7NxjC/view?usp=sharing)

**Comment by MANI14011998** [comment]
Ma'am i am working on this issue .

**Comment by sunjunkie** [comment]
@vatsalkul  Please submit the PR for your change.
